test/langtools/jdk/javadoc/doclet/testRecordTypes/TestRecordTypes.java

```diff
@@ -605,4 +605,41 @@ public void testSerializableType(Path base) throws IOException {
                     </ul>
                     </section>""");
     }
+
+    @Test
+    public void testPackageTree(Path base) throws IOException {
```

Comment:
Am I right saying that we cannot easily test the original bug symptom, which is the presence of "Direct Known Subclasses:" on the api/java.base/java/lang/Record.html page?

Reply:
Yes, that is a specific test on a specific page in the JDK API documentation.

A possible test would involve ensuring access to the `src/` directory, and then maybe running javadoc on a subset of APIs, such as `java.lang.*` with maybe dummy placeholders for JDK custom tags, like `@jls`, `@jvms`, `@moduleGraph` etc.